Mathlib/Algebra/BigOperators/Finprod.lean

```diff
@@ -949,6 +949,19 @@ lemma finprod_option {f : Option α → M} (hf : (mulSupport (f ∘ some)).Finit
   · rw [finprod_mem_range]
     exact Option.some_injective _
 
+@[to_additive]
+lemma finprod_mem_powerset_insert {f : Set α → M} {s : Set α} {a : α} (hs : s.Finite)
+    (has : a ∉ s) : ∏ᶠ t ∈ 𝒫 insert a s, f t = ∏ᶠ t ∈ 𝒫 s, f t * f (insert a t) := by
```

Comment:
The analogous `Finset` lemma, `Finset.sum_powerset_insert`, writes this as
```suggestion
    (has : a ∉ s) : ∏ᶠ t ∈ 𝒫 insert a s, f t = (∏ᶠ t ∈ 𝒫 s, f t) * ∏ᶠ t ∈ 𝒫 s, f (insert a t) := by
```
are you happy to make them match?